bioblend/galaxy/invocations/__init__.py

```diff
@@ -128,6 +131,100 @@ def show_invocation(self, invocation_id):
+                         inputs_by: Optional[str] = None, parameters_normalized: bool = False):
+        """
+        Rerun a workflow invocation. For more extensive documentation of all
+        parameters, see the ``gi.workflows.invoke_workflow()`` method.
+
+        :type invocation_id: str
+        :param invocation_id: Encoded workflow invocation ID to be rerun
+
+        :type inputs_update: dict
+        :param inputs_update: If different datasets should be used to the original
+          invocation, this should contain a mapping of workflow inputs to the new
+          datasets and dataset collections.
+
+        :type params_update: dict
+        :param params_update: If different non-dataset tool parameters should be
+          used to the original invocation, this should contain a mapping of the
+          new parameter values.
+
+        :type history_id: str
+        :param history_id: The encoded history ID where to store the workflow
+          outputs. Alternatively, ``history_name`` may be specified to create a
+          new history.
+
+        :type history_name: str
+        :param history_name: Create a new history with the given name to store
+          the workflow outputs. If both ``history_id`` and ``history_name`` are
+          provided, ``history_name`` is ignored. If neither is specified, a new
+          'Unnamed history' is created.
+
+        :type import_inputs_to_history: bool
+        :param import_inputs_to_history: If ``True``, used workflow inputs will
+          be imported into the history. If ``False``, only workflow outputs will
+          be visible in the given history.
+
+        :type allow_tool_state_corrections: bool
+        :param allow_tool_state_corrections: If True, allow Galaxy to fill in
+          missing tool state when running workflows. This may be useful for
+          workflows using tools that have changed over time or for workflows
+          built outside of Galaxy with only a subset of inputs defined.
+
+        :type replacement_params: dict
+        :param replacement_params: pattern-based replacements for post-job
+          actions
+
+        :type inputs_by: str
+        :param inputs_by: Determines how inputs are referenced. Can be
+          "step_index|step_uuid" (default), "step_index", "step_id", "step_uuid", or "name".
+
+        :type parameters_normalized: bool
+        :param parameters_normalized: Whether Galaxy should normalize the input
+          parameters to ensure everything is referenced by a numeric step ID.
+          Default is ``False``, but when setting parameters for a subworkflow,
+          ``True`` is required.
+
+        :rtype: dict
+        :return: A dict describing the new workflow invocation.
+
+        .. note::
+          This method can only be used with Galaxy ``release_21.05`` or later.
+        """
+        invocation_details = self.show_invocation(invocation_id)
+        workflow_id = invocation_details['stored_workflow_id']
```

Comment:
This shouldn't be part of the public API and I thought we agreed on that in https://github.com/galaxyproject/galaxy/pull/11881#discussion_r618314707

Reply:
And I see you've changed that in a later commit :)